bottle_app.py

```diff
@@ -629,14 +629,59 @@ def api_get_movie_trackpoints():
         labels         = sorted( ( tp['label'] for tp in trackpoint_dicts) )
         frame_dicts    = defaultdict(dict)
 
+        ruler0_x, ruler20_x = -1, -1
+        ruler0_y, ruler20_y = -1, -1
+        ruler0_frame_number = -1
+        sensor_width_mm = 36  # Sensor width 36 millimeters (this is a common value,
+        # but should be adjusted according to actual circumstances)
+        actual_distance_mm = 20
+        mm_per_inch = 25.4  # common value
+
         for tp in trackpoint_dicts:
-            frame_dicts[tp['frame_number']][tp['label']+' x'] = tp['x']
-            frame_dicts[tp['frame_number']][tp['label']+' y'] = tp['y']
```

Comment:
Here you are removing functionality that is useful. I do not think that you should be removing this functionality. I think that it is useful to provide the pixel x,y for each trackpoint. Please keep this code in.